dojo/templates/dojo/import_scan_results.html

```diff
@@ -108,6 +108,7 @@ <h4> JIRA </h4>
 	<li><b>ORT evaluated model Importer </b> - Import ORT evaluated model report in JSON format.</li>
 	<li><b>Outpost24 Scan</b> - Import Outpost24 endpoint vulnerability scan in XML format.</li>
 	<li><b>OpenVAS CSV</b> - Import OpenVAS Scan in CSV format. Export as CSV Results on OpenVAS.</li>
+	<li><b>OssIndex Devaudit SCA Scan</b> - Import OssIndex Devaudit SCA Scan in json format.</li>
```

Comment:
There is some inconsitancies in the PR for the values of parser identifier:
* `OssIndex Devaudit SCA Scan`
* `OssIndex DevAudit scan result importer`
* `OssIndex Devaudit SCA Scan Results`

Those 3 values should be the same.

Reply:
Fixed all three locations using "OssIndex Devaudit SCA Scan Importer"